packages/components/tests/dummy/translations/en-us.yaml

```diff
@@ -0,0 +1,11 @@
+# price_banner: The {product} costs {price, number, ::currency/USD}
```

Comment:
Once this is merged, please create a ticket (so we don't forget) in the new website implementation epic (https://hashicorp.atlassian.net/browse/HDS-680) about moving these new features to the new `website` app too.

Reply:
Decided to take this out for the time being.